CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Replace promtail occurrences by logging to be more generic.
```

Comment:
Why do we want to be more generic?

Reply:
Currently we have a mix of specific (promtail) and generic (logging).
Plus, configmap and secret are naming with a generic name.
Code and logs are more readable IMO with generic naming everywhere :-)